internal/util/text.go

```
@@ -0,0 +1,181 @@
+package util
+
+import (
+	"strings"
+	"unicode"
+
+	"golang.org/x/text/runes"
+	"golang.org/x/text/transform"
+	"golang.org/x/text/unicode/norm"
+)
+
+var ( //nolint:gochecknoglobals // Precompiled transforms/tables are read-only and reused for performance.
```

golangci-lint (2.2.2)

directive `//nolint:gochecknoglobals // Precompiled transforms/tables are read-only and reused for performance.` is unused for linter "gochecknoglobals"

(nolintlint)